### DISCUSSION TITLE
2025 Update

### DISCUSSION THREAD

**Discussion by magiccodingman** [magiccodingman/Magic.IndexedDb#18] [Announcements]
Shout out to @yueyinqiu for the multiple updates in 2024!

I started this project and made my last commit within a 13 day period in 2023. I did not realize anyone would see or care for the code, so thank you for the support!

I own a small software company and I try to separate code where I can to open source. As that's important to me personally. I've just not had the direction as of yet to come back to the project, but I have plans to be coming back in this direction in 2025.

Note this'll always be open source. My company aims to build software for a small niche field to support persons with disabilities. But I have a personal strong belief in open source, decentralization, and stuff of that sorts. So, I try to share what I can when I can.

But thanks to additional updates from others, this project has continued :) Enjoy the holidays everyone!

**Comment by yueyinqiu** [comment]
Really appreciate your spirit of open source. And your company and other personal affairs are also important. Heartily hope your company's projects could achieve success as well.
